webapp/src/app/shared/phaser/main-scene.ts

```diff
@@ -8,49 +8,55 @@ import { TileDrawer } from './tilemap/tile-drawer';
 import { EntityManager } from './entities/entity-manager';
 
 export class MainScene extends Phaser.Scene {
-
+	
 	private readonly borderSize = 1;
-
+	
 	private border?: Phaser.GameObjects.Rectangle;
 	private sub?: Subscription;
 	private borderVisible = true;
-
+	
 	constructor() {
-		super({ key: 'main' });
+		super({key: 'main'});
 	}
-
+	
 	preload() {
-
+		
 		this.load.image('pixel', 'assets/pixel.png');
-
+		
 		this.load.json('destructibles.json', 'assets/destructibles.json');
 		this.load.json('destructible-types.json', 'assets/destructible-types.json');
 		this.load.crossOrigin = 'anonymous';
-
+		
 		// this.load.on('progress', (val: number) => console.log(val));
-
+		
 		// this.load.maxParallelDownloads = this.res.images.length;
 		this.load.once('complete', () => Globals.globalEventsService.loadComplete.next());
 	}
-
+	
 	create() {
 		const game = this.game;
 		this.cameras.main.setBackgroundColor('#616161');
 		game.canvas.oncontextmenu = function (e) {
 			e.preventDefault();
 		};
-
+		
 		game.scale.scaleMode = Phaser.Scale.ScaleModes.NONE;
-
+		
 		const entityManager = new EntityManager(this, false);
-
+		
 		const tileMap = new CCMap(game, this, entityManager);
 		Globals.map = tileMap;
-
+		
 		this.sub = Globals.mapLoaderService.map.subscribe((map) => {
 			if (map) {
 				tileMap.loadMap(map);
 				this.rescaleBorder();
+				
+				// reset camera position on map load
+				const cam = this.cameras.main;
+				cam.scrollX = 0;
+				cam.scrollY = 0;
+				cam.zoom = 1;
```

Comment:
Possible improvement: center map in center of viewport instead of top left

Reply:
implemented it. 
It required changing the dimensions of the containing canvas element to accurately get the center. Could cause issues on different display resolutions, so it would be good if you could run it and check if the yellow borders are visible on the bottom or right like in that image.

![image](https://user-images.githubusercontent.com/9483499/119272979-15b76480-bc09-11eb-8c21-dac9b9db09f7.png)